app/case/server.py

```diff
@@ -64,3 +64,16 @@ def update_status(deed_id):
             return jsonify(case_status=case_status), status.HTTP_200_OK
         else:
             abort(status.HTTP_400_BAD_REQUEST)
+
+    @blueprint.route('/case/<case_id>/deed', methods=['POST'])
+    def update_case_deed(case_id):
+        case = Case.get(case_id)
+
+        if case is None:
+            abort(status.HTTP_404_NOT_FOUND)
+
+        deed_id = request.data['deed_id']
+        case.deed_id = deed_id
+
+        case.save()
+        return str(status.HTTP_200_OK)
```

Comment:
Yeah sure, if we're not hitting mega scale then that should be ok plus it gives confidence sign some of these API's will be used by externals.

if we're going to include it then yes we should return everything that has changed. perhaps only what has changed from other API's